react-frontend/src/components/Home/cardlist.js

```diff
@@ -76,7 +77,12 @@ const CardList = () => {
           </Col>
         </Row>
       </Grid>
-      <Grid style={{ paddingTop: '90px' }}>
+      <Grid style={{ paddingTop: '60px' }}>
+        <Row>
+          <Col sm={12}>
+            <Title3>Learn from Others</Title3>
```

Comment:
This is an h3 tag, I believe we want it to be an h2 to be inline with our accessibility standards

Reply:
Good call 👍 — just made a [new commit](https://github.com/bcgov/digital.gov.bc.ca/pull/426/commits/9a778e0ad980b13cdeed81803bff55fdc091ffbd) to replace that